index.js

```diff
@@ -201,18 +203,21 @@ var animationQueue = new AnimationQueue();
 // Give each notification a unique id
 var latestID = 0;
 
-function notify(title, text, url, image, onClickFunc, onShowFunc, onCloseFunc) {
+function notify(title, text, url, image, sound, onClickFunc, onShowFunc, onCloseFunc) {
```

Comment:
We can't add sound in the middle of this function. That would break API compatibility. You could add it at the end or increase the major version number and add a note to the READEM.md.